dashboard/app/dsl/base_dsl.rb

```diff
@@ -30,7 +30,9 @@ def parse_output
 
   # after parse has been done, this function returns a hash of all the user-visible strings from this instance
   def i18n_hash
-    {"en" => { "data" => { prefix => i18n_strings }}}
+    # Filter out any entries with nil key or value
+    hash = i18n_strings.select { |key, value| !key.nil? && !value.nil? }
```

Comment:
nit: I would prefer `key && value` if you're not worried about the false/nil thing (I think you shouldn't be because they are strings?)